nodes/Imap/operations/email/functions/EmailDownload.ts

```diff
@@ -0,0 +1,61 @@
+import { FetchQueryObject, ImapFlow } from "imapflow";
+import { IExecuteFunctions, INodeExecutionData } from "n8n-workflow";
+import { IResourceOperationDef } from "../../../utils/CommonDefinitions";
+import { getMailboxPathFromNodeParameter, parameterSelectMailbox } from "../../../utils/SearchFieldParameters";
+
+export const downloadOperation: IResourceOperationDef = {
+  operation: {
+    name: 'Download',
```

Comment:
Please rename to `Download as EML`, so it won't be confused with download attachments

Reply:
I think you already merged it and changed it yourself? Sry for the late response I was on a travel. 
And the other pull request is also merged

Reply:
> I think you already merged it and changed it yourself? Sry for the late response I was on a travel. And the other pull request is also merged

Yes, I made those changes and merged the pull request. Thanks again for your contribution!